pkg/types/v2/metadata.go

```diff
@@ -58,6 +62,20 @@ func (mt MetadataTest) Validate(channel chan interface{}) bool {
 	res := &types.TestResult{
 		Name: mt.LogName(),
 	}
+	if len(mt.Env) > 0 && len(mt.EnvVars) > 0 {
+		res.Error("Cannot use both env and envVars in the same test")
```

Comment:
Instead of raising an error, we can merge these two slices. They don't necessarily have to be mutually exclusive.